JudoPayDotNet/Http/VersioningHandler.cs

```diff
@@ -11,7 +11,7 @@ internal class VersioningHandler : DelegatingHandler
     {
         public const string API_VERSION_HEADER = "api-version";
 
-        internal const string DEFAULT_API_VERSION = "6.13";
+        internal const string DEFAULT_API_VERSION = "6.15";
```

Comment:
https://thedojo.atlassian.net/browse/JR-6130 will be added in 6.16 (not yet released)